i18n/validate.py

```diff
@@ -46,7 +46,7 @@ def msgfmt_check_po_file(filename):
     """
     # Use relative paths to make output less noisy.
     rfile = os.path.relpath(filename, config.LOCALE_DIR)
-    out, err = call('msgfmt -c {}'.format(rfile), working_directory=config.LOCALE_DIR)
+    out, err = call('msgfmt -c -o /dev/null {}'.format(rfile), working_directory=config.LOCALE_DIR)
```

Comment:
The -o switch indicates where to leave the resulting message.mo file.  This command is being run only for its diagnostic output, which still appears.